arcade/draw_commands.py

```diff
@@ -617,11 +618,18 @@ def draw_triangle_outline(x1: float, y1: float,
 # --- BEGIN RECTANGLE FUNCTIONS # # #
 
 
+@warning(
+    warning_type=NameChangeWarning,
+    new_name="draw_lrbt_rectangle_outline"
+)
 def draw_lrtb_rectangle_outline(left: float, right: float, top: float,
                                 bottom: float, color: Color,
                                 border_width: float = 1):
     """
-    Draw a rectangle by specifying left, right, top, and bottom edges.
+    Draw a rectangle by specifying left, right, top and bottom edges.
+
+    .. deprecated:: 3.0
+        Use :py:func:`draw_lrbt_rectangle_outline` instead!
```

Comment:
Although this may build without perfect alignment, it's a good habit to be precise. Sphinx is picky about whitespace like Python is, just slightly less so.
```suggestion
       Use :py:func:`draw_lrbt_rectangle_outline` instead!
```